custom_components/tuya_custom/tuyaha/devices/climate.py

```diff
@@ -110,7 +111,12 @@ def support_humidity(self):
             return False
 
     def turn_on(self):
-        self.api.device_control(self.obj_id, "turnOnOff", {"value": "1"})
+        if self._control_device("turnOnOff", {"value": "1"}):
+            self._update_data("state", True)
 
     def turn_off(self):
-        self.api.device_control(self.obj_id, "turnOnOff", {"value": "0"})
+        if self._control_device("turnOnOff", {"value": "0"}):
+            self._update_data("state", False)
```

Comment:
I think that here is used "false" based on original library

Reply:
👍